src/program.js

```diff
@@ -341,6 +341,15 @@ Example: $0 --help run.
         demand: false,
         type: 'string',
       },
+      'firefox-port': {
+        alias: 'P',
+        describe: 'Attach to a running Firefox on this developer tools ' +
```

Comment:
This should explain things more from the perspective of what web-ext will do. Maybe something like "Connect to this debugger port on a running Firefox instance to install the browser extension."